packages/autorest.typescript/test/smoke/generated/arm-package-deploymentscripts-2019-10-preview/package.json

```diff
@@ -50,7 +50,6 @@
     "src/**/*.ts",
     "README.md",
     "LICENSE",
-    "rollup.config.js",
```

Comment:
Is the deletion of this line expected in non js-repo?

Reply:
Yes, there's no value in packing rollup config file.  it's for dev environment only.